inc/util/string_utils.hpp

```diff
@@ -46,6 +46,21 @@ namespace ebi
 
         output.swap(ret);
     }
+
+    /**
+     * Temporal implementation for mismatch with 2 starts and 2 ends. It is not in STL for c++11, but it will in c++14.
```

Comment:
From what I see in [Clang](http://clang.llvm.org/cxx_status.html#cxx14) and [GCC](https://gcc.gnu.org/projects/cxx-status.html#cxx14)'s standard support documentation, all the compilers we test our code against should be able to handle C++14. Maybe we could consider an upgrade in next release?